scripts/build.js

```diff
@@ -0,0 +1,70 @@
+import fs from "fs";
+import replaceInFiles from "replace-in-files";
+
+const transpiledPathPrefix = "./dist";
+
+async function build() {
+  await replaceInFiles({
+    files: [`${transpiledPathPrefix}/**/*.jsx`],
+    from: /export\s+default\s+function[^(]*\((.*)/gms,
+    to: (_match, rest) =>
+      `function MainComponent(${rest}\nreturn MainComponent(props, context);`,
+  });
+
+  await replaceInFiles({
+    files: [`${transpiledPathPrefix}/**/*.jsx`],
+    from: /^export /,
+    // NOTE: Empty string is ignored, so we use a function workaround it
+    to: () => "",
+  });
+
+  // WARNING: Don't allow "imports" in includes as this may lead to undefined
+  // behavior as replacements are done in parallel and one file may be getting
+  // replacements saved while the other file needs to include it, which ends up
+  // with empty content includes.
+  await new Promise((resolve) => {
+    fs.rename(
+      `${transpiledPathPrefix}/includes`,
+      `${transpiledPathPrefix}/../includes`,
+      () => {
+        resolve();
+      }
+    );
+  });
+
+  await replaceInFiles({
+    files: [`${transpiledPathPrefix}/**/*.jsx`],
+    from: /import .* from "@\/includes\/([^"]*)";/gms,
+    to: (_match, importPath) => {
+      const importedFileContent = fs.readFileSync(
+        `${transpiledPathPrefix}/../includes/${importPath}.jsx`,
+        "utf8"
+      );
+      return `/* INCLUDE: "includes/${importPath}.jsx" */\n${importedFileContent}/* END_INCLUDE: "includes/${importPath}.jsx" */`;
+    },
+  });
+
+  const packageJson = JSON.parse(
+    fs.readFileSync(new URL("../package.json", import.meta.url))
+  );
+
+  // await replaceInFiles({
+  //   files: [`${transpiledPathPrefix}/**/*.jsx`],
+  //   from: /^/m,
+  //   to: `/*\nLicense: ${packageJson.license}\nAuthor: ${packageJson.author}\nHomepage: ${packageJson.homepage}\n*/\n`,
+  // });
```

Comment:
I like this concept of stamping attributions and I think we should keep it; although I wonder if we could save it to metadata instead?

Maybe a feature saved for a [later iteration](https://github.com/NEARBuilders/bos-workspace/issues/2)

Reply:
Good point about metadata. Let's do that.

Reply:
Will keeps the comments for later use